### DISCUSSION TITLE
Welcome to react-collapse-pane Discussions!

### DISCUSSION THREAD

**Discussion by b-zurg** [b-zurg/react-collapse-pane#44] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aishwaryakakade** [comment]
How can I control the collapse button programatically ?
